config/config/common-config.yaml

```
@@ -853,6 +865,29 @@ trivy:
     interval: 5m
   tolerations: []
   affinity: {}
+
+  # configurations for an offline / air-gapped environment
+  # ref: https://github.com/aquasecurity/trivy/tree/main/helm/trivy#parameters
+  offline:
+    offlineScan: false
```

```suggestion
  offlineScan:
    enabled: false
```
Or is there other conditions when we would like to edit configuration under `offline` that do not require `offlineScan: true`? Else we should rename the section, and rename the enable flag to `offlineScanEnabled` or similar.

---

Maybe we should call the `offline` key `scanner` instead then, as this mainly applies to the scanner jobs, and the have it as `offlineScanEnabled` to match other config keys.